api/base/schemas/utils.py

```diff
@@ -29,10 +29,11 @@ def validate_dates(history):
     if history.get('start_date'):
         start_date = datetime.strptime(history['start_date'], '%Y-%m-%d')
 
-    if not history.get('ongoing', False):
-        if history.get('end_date'):
-            end_date = datetime.strptime(history['end_date'], '%Y-%m-%d')
-
-        if history.get('start_date') and history.get('end_date'):
-            if (end_date - start_date).days <= 0:
-                raise InvalidModelValueError(detail='End date must be greater than or equal to the start date.')
+        if not history.get('ongoing', False):
```

Comment:
Since we're here, could you make this a positive `if` rather than the negative case, so it's a bit more readable? So `if history.get('ongoing', False):` and do the other branch, then put this branch in the else?

Reply:
Modified and changes have been tested locally